features/step_definitions/hardship_claim_steps.rb

```diff
@@ -0,0 +1,20 @@
+Then(/^I should be on the litigator new hardship claim page$/) do
```

Comment:
i'd call this file `litigator_hardship_claim_steps.` before getting into master. i have done this with`advocate_hardship_claim_steps`